actionmailer/test/delivery_methods_test.rb

```diff
@@ -102,16 +102,21 @@ def welcome(hash={})
   end
 
   test "ActionMailer should be told when Mail gets delivered" do
-    DeliveryMailer.expects(:deliver_mail).once
-    DeliveryMailer.welcome.deliver_now
+    DeliveryMailer.delivery_method = :test
```

Comment:
By default `delivery_method` is `smtp`, so it's trying to deliver email. I'm getting the following error because of it
````bash
MailDeliveryTest#test_ActionMailer_should_be_told_when_Mail_gets_delivered:
Errno::ECONNREFUSED: Connection refused - connect(2) for "localhost" port 25
````
So either we have to set `delivery_method` :test or we have to stub any instance `Mail::SMTP`.
WDYT?